elm/ES/UI/Color.elm

```diff
@@ -0,0 +1,98 @@
+module ES.UI.Color exposing (Color, black, blue, green, grey, orange, purple, red, teal, transparent, white, yellow)
```

Comment:
this file is generated, correct? Never to be touched manually -- Having this file here makes it diffiuclt to understand that it shouldn't be edited directly. I would expect that as part of webpack or something, this file would generate -- or that this would instaed be an NPM package that generates the elm file. 
I might be missing somethign though.

Reply:
It's generated via `generate_colors.rb`. I suppose we could do an after install hook in package.json?

Or I could simply generate a comment saying this is generated.